velox/exec/HashTable.h

```diff
@@ -89,6 +89,11 @@ class BaseHashTable {
   struct RowsIterator {
     int32_t hashTableIndex_{-1};
     RowContainerIterator rowContainerIterator_;
+
+    void reset() {
```

Comment:
Just `lastProbeIterator_ = {}` should enough?

Reply:
I think reset is better then assignment.

Reply:
You would need to repeat the default value in two places.  Does not hurt too much for a small class like this though.
There is a way to get best of both worlds: `void reset() { *this = {}; }`.

Reply:
I mean reset() is more clear in such use case than an assignment to default. Yeah, we could implement reset as *this = {}. Will update. Thanks!